cfgov/unprocessed/css/organisms/bureau-structure.less

```diff
@@ -9,10 +9,10 @@
 @node-width__desktop: 405px;
 @node-width__tablet: 300px;
 @node-width__mobile: none;
-@node-border-width: 3px;
+@node-border-width: 1px;
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2532/commits/861674d42470d1ebe15c3ac204e66b4fa58f741c.